### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by GregoireHebert**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #31
| License       | MIT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,26 +4,44 @@
...
-
-DocuSign also offers the possibility to sign remotely, or to trigger a simple agreement click.
-But these options are not available yet.
-Feel free to contribute :)
+This Symfony Bundle is used to create electronic signature with DocuSign.
+An Electronic Signature ensure a person agreed with the document.
+
+1.  [Requirements](#requirements)
+1.  [Bundle Installation](#bundle-installation)
+    1.  [register the bundle](#register-the-bundle)
```
Comment (vincentchalamon) [inline README.md]:
```suggestion
    1.  [Register the bundle](#register-the-bundle)
```

File: README.md [hunk 2]
```diff
@@ -4,26 +4,44 @@
...
+    1.  [Configure the bundle](#configure-the-bundle)
+1.  [Docusign Configuration](#docusign-configuration)
```
Comment (vincentchalamon) [inline README.md]:
```suggestion
1.  [DocuSign configuration](#docusign-configuration)
```

File: README.md [hunk 3]
```diff
@@ -4,26 +4,44 @@
...
+    1.  [Types of signatures](#types-of-signatures)
+    1.  [Format Restrictions:](#format-restrictions)
```
Comment (vincentchalamon) [inline README.md]:
```suggestion
    1.  [Format restrictions:](#format-restrictions)
```

File: README.md [hunk 4]
```diff
@@ -4,26 +4,44 @@
...
+    1.  [Add an integration key on docusign](#add-an-integration-key-on-docusign)
```
Comment (vincentchalamon) [inline README.md]:
```suggestion
    1.  [Add an integration key on DocuSign](#add-an-integration-key-on-docusign)
```

File: README.md [hunk 5]
```diff
@@ -4,26 +4,44 @@
...
+    1.  [Configure a custom storage without league/flysystem-bundle](#configure-a-custom-storage-without-leagueflysystem-bundle)
+    1.  [Use league/flysystem-bundle](#use-league/flysystem-bundle)
+        1.  [Installation](#installation)
+        1.  [Configure the storage](#configure-the-storage)
+    1.  [Override configuration signature positions](#override-configuration-signature-positions)
+    1.  [Events](#events)
+        1. [Sign events](#sign-events)
+        1. [Webhook events](#webhook-events)
+    1.  [Using an external route as callback](#using-an-external-route-as-callback)
+1.  [Backward Compatibility promise](#backward-compatibility-promise)
```
Comment (vincentchalamon) [inline README.md]:
```suggestion
1.  [Backward compatibility promise](#backward-compatibility-promise)
```

Comment (GregoireHebert) [reply README.md]: I want to keep the uppercase here as it refers to the BC accronym

File: composer.json [hunk 6]
```diff
@@ -1,7 +1,27 @@
...
+    "authors": [
+        {
+            "name": "Grégoire Hébert",
+            "email": "gregoire@les-tilleuls.coop",
+            "homepage": "https://knot.gheb.dev/",
+            "role": "Developer"
+        },
+        {
+            "name": "Vincent Chalamon",
+            "email": "vincent@les-tilleuls.coop",
```
Comment (vincentchalamon) [inline composer.json]:
```suggestion
            "email": "vincentchalamon@gmail.com",
```

**Pull request merged**